src/schema/datatypes/meg.yaml

```diff
@@ -0,0 +1,50 @@
+group1:
+  datatypes:
+    - meg
+  suffices:
+    - meg
+  extensions:
+    - json
+    - ctf/
+    - fif
+    - 4d/
```

Comment:
`<rant>` wow, I didn't know that (sub)folders are allowed here.   I truly think this was a step backward for meeg proposal to just allow all those formats -- there is no 'standardization' now, tools will just support some but not the other formats etc. `</rant>`